addons/hr_attendance/static/src/components/manual_selection/manual_selection.xml

```diff
@@ -17,9 +17,9 @@
     </t>
     <t t-name="hr_attendance.public_kiosk_manual_selection">
         <t t-if="!this.props.displayBackButton">
-            <button t-on-click="() => this.props.onClickBack()" class="o_hr_attendance_back_button o_hr_attendance_back_button_md btn btn-secondary d-none d-md-inline-flex align-items-center position-absolute top-0 start-0 rounded-circle">
-            <i class="oi fa-2x fa-fw oi-chevron-left me-1" role="img" aria-label="Go back" title="Go back"/>
-        </button>
+            <button t-on-click="() => this.props.onClickBack()" class="o_hr_attendance_back_button o_hr_attendance_back_button_md btn btn-secondary d-none d-md-inline-flex align-items-center position-absolute top-0 start-0 rounded-circle" style="margin-left: 50px; margin-top: 50px;">
+                <i class="oi fa-2x fa-fw oi-chevron-left me-1" role="img" aria-label="Go back" title="Go back"/>
+            </button>
```

Comment:
it's better to use Bootstrap classes like `ml-` and `mt-` instead of setting styles manually.